openff/interchange/_tests/unit_tests/smirnoff/test_create.py

```diff
@@ -231,6 +234,98 @@ def test_charges_from_molecule_reordered(
 
         assert numpy.allclose(expected_charges, found_charges)
 
+    def test_error_when_any_missing_partial_charges(self, sage):
+        topology = Topology.from_molecules(
+            [
+                Molecule.from_smiles("C"),
+                Molecule.from_smiles("CCO"),
+            ],
+        )
```

Comment:
Should this topology include a molecule *with* partial charges, to ensure that it's testing if there are *any* missing charges?